ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx

```diff
@@ -285,30 +326,45 @@ function VulnReportsPage() {
                                                 reportId: report.id,
                                             }
                                         ) as string;
+                                        const isReportStatusPending = !!(
+                                            report.reportSnapshot?.reportStatus.runState ===
+                                                'PREPARING' ||
+                                            report.reportSnapshot?.reportStatus.runState ===
+                                                'WAITING'
+                                        );
```

Comment:
Nit: is the `!!` necessary here? It looks like the value returned by the expression is already a `boolean`.

Reply:
Yea, that's true. I'll change it. Eslint changed it to that when I wrote it so I just kept it.